cmake/Codegen.cmake

```diff
@@ -69,6 +69,21 @@ if(INTERN_BUILD_ATEN_OPS)
 
   file(GLOB_RECURSE all_python "${CMAKE_CURRENT_LIST_DIR}/../torchgen/*.py")
 
+  # Rowwise scaling requires sm_90a gencode
+  if(USE_CUDA)
```

Comment:
We should do it only if CUDA_ARCH_FLAGS already include `compute_90` (otherwise it fails to compile with 11.8)